### PR TITLE
Fixed: package meta for curseforge

### PR DESCRIPTION
**Pull request by JanGalek**
~~also added submodule init for make pull (we need clone it)~~

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,7 @@ all:
 
 pull:
 	git pull
+	git submodule init
```
Comment (idiomatic) [inline Makefile]: Oops.  I got spoiled by GitHub Actions `submodules: true`.

**Review by idiomatic — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (JanGalek) [reply Makefile]: oooh, I just tested it on my local machine, sorry :)

Comment (idiomatic) [reply Makefile]: On second thought, the `git submodule init` was appropriate with the `pull` target... but not with the replacement `update` target.  Git setup is an external problem to local builds.

Comment (JanGalek) [reply Makefile]: commit dropped ;)

**Pull request merged**